platform-sdk/swirlds-config-processor/src/main/java/com/swirlds/config/processor/antlr/AntlrConfigRecordParser.java

```diff
@@ -151,22 +150,44 @@ private static ConfigDataRecordDefinition createDefinition(
             @NonNull final String packageName,
             @NonNull final List<String> imports) {
         final String recordName = recordContext.identifier().getText();
-        final String configPropertyNamePrefix = getConfigDataAnnotationValue(recordContext, packageName, imports);
-        final Map<String, String> javadocParams = unitContext.children.stream()
-                .filter(c -> AntlrUtils.isJavaDocNode(c))
-                .map(c -> c.getText())
-                .map(t -> AntlrUtils.getJavaDocParams(t))
-                .reduce((m1, m2) -> {
-                    m1.putAll(m2);
-                    return m1;
-                })
-                .orElse(Map.of());
-        final Set<ConfigDataPropertyDefinition> propertyDefinitions =
-                recordContext.recordHeader().recordComponentList().recordComponent().stream()
-                        .map(c -> createPropertyDefinition(
-                                c, configPropertyNamePrefix, packageName, imports, javadocParams))
-                        .collect(Collectors.toSet());
-        return new ConfigDataRecordDefinition(packageName, recordName, configPropertyNamePrefix, propertyDefinitions);
+
+        final Class<ConfigData> configDataAnnotation = ConfigData.class;
+        final Class<ConfigProperty> configPropertyAnnotation = ConfigProperty.class;
```

Comment:
Consider using constants

Reply:
Hey Timo, given that, in a way, calling `.class` can be considered a constant reference, I don't see much gain by extracting it or even by having it extracted to a variable.
Also given that the possible annotation wont have the same structure, many of the changes I did don't make sense.

Reply:
I didn’t expect gains form that change but for me this is a constant that belongs to the top but fine with leaving it as it is